librz/core/cfile.c

```diff
@@ -1115,16 +1149,16 @@ RZ_API RzCoreFile *rz_core_file_open_many(RzCore *r, const char *file, int perm,
 		RzCoreFile *fh = RZ_NEW0(RzCoreFile);
 		if (fh) {
 			fh->core = r;
-			fh->fd = fd->fd;
+			fh->fd = desc->fd;
 			r->file = fh;
 			rz_list_append(r->files, fh);
-			rz_core_bin_load(r, fd->name, loadaddr);
+			rz_core_bin_load(r, desc->name, loadaddr);
 		}
 	}
 	return NULL;
 }
 
-/* loadaddr is r2 -m (mapaddr) */
+/* loadaddr is rizin -m (mapaddr) */
 RZ_API RzCoreFile *rz_core_file_open(RzCore *r, const char *file, int flags, ut64 loadaddr) {
```

Comment:
Please add `RZ_NONNULL` for `file`. Ideally, doxygen as well, while you are on it.